pom.xml

```diff
@@ -26,13 +26,12 @@
                             <goal>npl-test</goal>
                             <goal>npl-api</goal>
                         </goals>
-                        <configuration>
-                            <enableCompilerOptimisations>true</enableCompilerOptimisations>
-                            <sources>${project.basedir}/src/main/npl-1.0</sources>
-                            <testSources>${project.basedir}/src/test/npl</testSources>
-                        </configuration>
                     </execution>
                 </executions>
+                <configuration>
+                    <sources>${project.basedir}/src/main/npl-1.0</sources>
+                    <testSources>${project.basedir}/src/test/npl</testSources>
+                </configuration>
```

Comment:
This makes it a global plugin configuration rather than being bound to the build lifecycle.